dnf-behave-tests/features/vars-repo_id.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Subtitute variables in repo id
+
+
+@bz1748841
+Scenario: Variables without {} are substituted in repo id
+  Given I create file "/etc/dnf/vars/distrib" with
+    """
+    fedora
+    """
+    And I create file "/etc/yum.repos.d/dnf-ci-test.repo" with
+    """
+    [dnf-ci-test-$distrib]
+    name=dnf-ci-test-$distrib test repository
+    enabled=0
+    """
+  When I execute dnf with args "repolist --disabled"
+  Then the exit code is 0
```

Comment:
The above two lines are under-indented by a space. I quite dislike how fiddly this alignment style is, but lets try to keep it consistent... Same for the second test below.